ci/playbooks/kuttl/deploy-deps.yaml

```diff
@@ -89,35 +89,15 @@
           ignore_errors: true
           register: openstack_operator_subscription
 
-        - name: Install openstack operator
+        - name: Install openstack operator and wait for the csv to succeed
           when: openstack_operator_subscription.stdout == ""
           vars:
-            make_openstack_env: "{{ cifmw_edpm_prepare_common_env |
+            make_openstack_init_env: "{{ cifmw_edpm_prepare_common_env |
               combine(cifmw_edpm_prepare_make_openstack_env) }}"
-            make_openstack_dryrun: "{{ cifmw_edpm_prepare_dry_run }}"
+            make_openstack_init_dryrun: "{{ cifmw_edpm_prepare_dry_run }}"
           ansible.builtin.include_role:
             name: 'install_yamls_makes'
-            tasks_from: 'make_openstack'
-
-        - name: Wait for OpenStack subscription creation
-          when: openstack_operator_subscription.stdout == ""
-          ansible.builtin.command:
-            cmd: >-
-              oc get sub openstack-operator
-              --namespace=openstack-operators
-              -o=jsonpath='{.status.installplan.name}'
-          register: cifmw_edpm_prepare_wait_installplan_out
-          until: cifmw_edpm_prepare_wait_installplan_out.rc == 0 and cifmw_edpm_prepare_wait_installplan_out.stdout != ""
-          retries: 30
-          delay: 10
-
-        - name: Wait for OpenStack operator to get installed
-          when: openstack_operator_subscription.stdout == ""
-          ansible.builtin.command:
-            cmd: >-
-              oc wait InstallPlan {{ cifmw_edpm_prepare_wait_installplan_out.stdout }}
-              --namespace=openstack-operators
-              --for=jsonpath='{.status.phase}'=Complete --timeout=20m
+            tasks_from: 'make_openstack_init'
```

Comment:
the commit message references make openstack_wait but this is calling _init

Reply:
ah so _init calls _wait
https://github.com/openstack-k8s-operators/install_yamls/blob/26997dbd080199841599b5ca7b3c47830769fd4e/Makefile#L746

Reply:
initially I tried with `openstack_wait`, but that was not enough, and I forgot to update the commit msg, updated now, thanks!

Reply:
thanks